test/core/test_sympy_python_convert.py

```diff
@@ -42,10 +42,10 @@ def compare(self, mathics_expr, sympy_expr, **kwargs):
         self.compare_to_mathics(mathics_expr, sympy_expr)
 
     def testSymbol(self):
-        self.compare(Symbol("Global`x"), sympy.Symbol("_Mathics_User_Global`x"))
+        self.compare(Symbol("Global`x"), sympy.Symbol("_mg`x"))
         self.compare(
-            Symbol("_Mathics_User_x"),
-            sympy.Symbol("_Mathics_User_System`_Mathics_User_x"),
+            Symbol("_mu_x"),
+            sympy.Symbol("_mu`x"),
```

Comment:
I would expect that if the conversion routines haven't changed,  and the prefix is `_m_`, 
```"_Mathics_User_System`_Mathics_User_x"``` should be now ```"_m_System`_m_x"```,
with the prefix both in the symbol name and the context. 
In any case, it would be reasonable to put the prefix in the context.

Reply:
See above. Not sure why testing works when it shouldn't. Testing is probably a bit weak then.

The issue of shortening context prefixes will be addressed in a more isolated PR.

Reply:
good. Then let's merge this when you feel it is ready.

Reply:
The reason why the tests are "weak" is that we are not checking the result of the comparison. I will put a fix for this in another PR.